src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -177,7 +177,7 @@ describe('StoryPromo Container', () => {
     });
 
     it('should render relative time if timestamp < 10 hours', () => {
-      const oneMinuteAgo = Math.floor(Date.now() / 1000) - 60;
+      const oneMinuteAgo = Math.floor(Date.now()) - 60 * 1000;
```

Comment:
You won't need Math.floor if there's no division :)
```suggestion
      const oneMinuteAgo = Date.now() - 60 * 1000;
```